docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -1,90 +1,84 @@
 
 -   Sequences of **duplicate code** (clones). These are the same ones you find in the [Quality Duplication tabs](../repositories/pull-requests.md#duplication-tabs) in the Codacy app.
 
 -   Variations in **code complexity**.
 
-## Navigating to found problems
+## Reviewing pull request issues
 
-To navigate to a found problem:
+In the Problems tab, Codacy displays the same Quality issues you find in the main view and lets you navigate to the exact line of code where the issue was found.
 
-1.  Open the **Problems tab** (use `Ctrl+Shift+M` on Windows/Linux or `Cmd+Shift+M` on macOS)
-
-1.  Click the name of the issue you wish to review.
+!!! note
+    Code coverage, duplicates, and complexity aren't currently shown in the Problems tab.
 
-    Line-based navigation is currently only supported for Quality issues.
+To review Quality issues:
 
-    ![Navigate to a specific issue from the Problems tab](images/codacy-vscode-extension-problems-tab.png)
+1.  Open the **Problems tab** (use `Ctrl+Shift+M` on Windows/Linux or `Cmd+Shift+M` on macOS).
 
-1.  Hover over a highlighted issue in the code editor to view more details and available quick fixes.
+1.  Click the name of the issue you want to review.
 
-    For a list of tools that support quick fixes, see [Supported languages and tools](./supported-languages-and-tools.md#supported-languages-and-tools)
+    ![Navigate to a specific issue from the Problems tab](images/codacy-vscode-extension-problems-tab.png)
 
-## Refreshing pull request data
+1.  Hover over a highlighted issue in the code editor to view available actions and suggested quick fixes (if available).
 
-The Codacy extension refreshes the pull request data every time you push new changes to the Git provider, after the analysis is complete. The analysis usually takes less than 5 minutes.
+    For a list of tools that support quick fixes, see [Supported languages and tools](./supported-languages-and-tools.md#supported-languages-and-tools).
 
-To refresh the pull request data manually, click the Refresh Pull Request button in the main view.
+1.  Once you've addressed the problems in your code, push your changes to the Git provider so that Codacy analyzes the updated code.
 
-## Troubleshooting
+When the analysis is complete, the Codacy extension automatically refreshes the pull request analysis result. You can also refresh the pull request data manually by clicking the Refresh Pull Request button in the main view.
 
-### <span class="skip-vale">Could not</span> find repository
+## See also
 
-If you encounter this error, confirm that the repository has been [added to Codacy Cloud](../organizations/managing-repositories.md#adding-a-repository) and that you have at least [Repository Read permissions](../organizations/roles-and-permissions-for-organizations.md) for the repository.
+-   [Troubleshooting the Codacy VS Code extension](https://marketplace.visualstudio.com/items?itemName=codacy-app.codacy#troubleshooting)
```

Comment:
@nicklem Don't we want to link to the [GitHub project](https://github.com/codacy/codacy-vscode-extension#troubleshooting) instead? 🤔

Reply:
That's also an option. It depends if we want to keep the users on the front-facing pages vs. the Git repository. We can update this in a follow-up.